examples/unconditional_image_generation/train_unconditional.py

```diff
@@ -467,7 +475,7 @@ def load_model_hook(models, input_dir):
     # Preprocessing the datasets and DataLoaders creation.
     augmentations = transforms.Compose(
         [
-            transforms.Resize(args.resolution, interpolation=transforms.InterpolationMode.BILINEAR),
+            transforms.Resize(args.resolution, interpolation=PIL.Image.BICUBIC),
```

Comment:
I believe this modification is only needed for `torch<=1.7`. I reverted this change and the script run fine.

Not to mention bilinear and bicubic are two different interpolation modes...

Reply:
Thanks for letting me know! I've updated the code to reverted back to the old interpolation mode.